terminal.go

```diff
@@ -494,6 +494,16 @@ func (g *LightningTerminal) startSubservers() error {
 		return err
 	}
 
+	// Collect the tags that LND was built with.
+	version, err := g.lndClient.Versioner.GetVersion(ctxc)
```

Comment:
The version RPC is already called on init and cached in `g.lndClient.Version`, we can use that directly.

Reply:
oh cool! ok does this mean I can also remove the `verrpc` tag from the readme? sounds like LND is automatically compiled with that tag?

Reply:
Ah, yes. I forgot that some subservers are always active and not behind a build tag. I guess only those that we declare as "experimental" still have a flag.